ci/resources/setup_ltp.sh

```diff
@@ -24,6 +24,11 @@ then
     cp -f manifest_Almalinux.template manifest.template
 fi
 
+if [[ "$base_os" == *"rocky"* ]]
+then
+    cp -f manifest_Rockylinux.template manifest.template
```

Comment:
Almalinux manifest file and rockylinux manifest file are same, we should use almalinux manifest file here